InterfaceFortran/TasmanianSG.f90

```diff
@@ -702,7 +702,7 @@ subroutine tsgGetComplexHierarchicalCoefficientsStatic(gridID, c)
   if ( tsgIsFourier(gridID) ) then
     call tsgghc(gridID, c_real)
     do i = 1,size(c)
-      c(i) = complex( c_real(i), c_real(i + size(c)) )
+      c(i) = cmplx( c_real(i), c_real(i + size(c)) )
```

Comment:
`c(i) = cmplx( c_real(i), c_real(i + size(c)), 8 )`

see comment before, this defaults to single-precision